packages/blade/src/utils/mapProps/mapProps.web.test.ts

```diff
@@ -0,0 +1,51 @@
+import mapProps from './mapProps';
+
+describe('mapProps', () => {
+  it('should work', () => {
+    expect(
+      mapProps({
+        accessibilityLabel: 'hello world',
+        accessibilityLabelledBy: 'id1',
+        accessibilityRole: 'button',
+      }),
+    ).toStrictEqual({
+      'aria-labelledby': 'id1',
+      'aria-label': 'hello world',
+      role: 'button',
+    });
+
+    expect(mapProps({ accessibilityLiveRegion: 'polite' })).toStrictEqual({
```

Comment:
can have another `it` which can have title 

`should return correct accessibility attributes for accessibilityLiveRegion`